_build_cfg.yml

```diff
@@ -83,8 +83,8 @@ Topics:
 Name: Using OpenShift
 Dir: using_openshift
 Topics:
-  - Name: Overview
-    File: overview
+#  - Name: Overview
+#    File: overview
   - Name: Command Line Interface
```

Comment:
The Welcome page (`welcome/index`) links to `using_openshift/overview`, so if we're going to remove this then the Welcome page should be updated, too. Should probably discuss with @CowboysFan.